[Jenkins Keycloak CI] - Cookies Tests - KcSamlBrokerTest

### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

ci

### Describe the bug

```
[ERROR] org.keycloak.testsuite.broker.KcSamlBrokerTest.loginWithIdpEntityIdCorrect  Time elapsed: 10.244 s  <<< FAILURE!
java.lang.AssertionError: 

Expected: response status code matches is <200>
     but: was <HttpResponseProxy{HTTP/1.1 400 Bad Request [content-length: 3390, Content-Language: en, Content-Security-Policy: frame-src 'self'; frame-ancestors 'self'; object-src 'none';, Content-Type: text/html;charset=utf-8, Referrer-Policy: no-referrer, Strict-Transport-Security: max-age=31536000; includeSubDomains, X-Content-Type-Options: nosniff, X-Frame-Options: SAMEORIGIN, X-Robots-Tag: none, X-XSS-Protection: 1; mode=block] ResponseEntityProxy{[Content-Type: text/html;charset=utf-8,Content-Length: 3390,Chunked: false]}}> with entity <!DOCTYPE html>
<html class="login-pf">
...
```

### Version

latest

### Regression

- [ ] The issue is a regression

### Expected behavior

-

### Actual behavior

-

### How to Reproduce?

`mvn -f testsuite/integration-arquillian/tests/base/pom.xml clean install -Pauth-server-quarkus -Pfirefox-strict-cookies -Dauth.server.host=auth-server-127-0-0-1.nip.io -Dauth.server.host2=127.0.0.1.sslip.io -Dtest=KcSamlBrokerTest#loginWithIdpEntityIdCorrect`

### Anything else?

_No response_

--- a/testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/broker/KcSamlBrokerTest.java
+++ b/testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/broker/KcSamlBrokerTest.java
@@ -63,6 +63,7 @@ import static org.keycloak.testsuite.util.SamlStreams.assertionsUnencrypted;
 import static org.keycloak.testsuite.util.SamlStreams.attributeStatements;
 import static org.keycloak.testsuite.util.SamlStreams.attributesUnecrypted;
 import static org.keycloak.testsuite.util.Matchers.bodyHC;
+import static org.keycloak.testsuite.util.ServerURLs.AUTH_SERVER_HOST2;
 
 /**
  * Final class as it's not intended to be overriden. Feel free to remove "final" if you really know what you are doing.
@@ -370,7 +371,7 @@ public final class KcSamlBrokerTest extends AbstractAdvancedBrokerTest {
     public void loginWithIdpEntityIdCorrect() throws Exception {
       // Set the expected IDP Entity ID to the correct value
       try (Closeable idpUpdater = new IdentityProviderAttributeUpdater(identityProviderResource)
-          .setAttribute(SAMLIdentityProviderConfig.IDP_ENTITY_ID, "https://localhost:8543/auth/realms/provider")
+          .setAttribute(SAMLIdentityProviderConfig.IDP_ENTITY_ID, "https://" + AUTH_SERVER_HOST2 + ":8543/auth/realms/provider")
           .update())
       {
         AuthnRequestType loginRep = SamlClient.createLoginRequestDocument(AbstractSamlTest.SAML_CLIENT_ID_SALES_POST + ".dot/ted", getConsumerRoot() + "/sales-post/saml", null);

